query.go

```diff
@@ -98,12 +107,92 @@ func (u *UtxorpcClient) GetUtxoByRef(
 		Index: txIndex,
 	}
 	req := &query.ReadUtxosRequest{Keys: []*query.TxoRef{txoRef}}
-	return u.ReadUtxos(req)
+	return u.ReadUtxosWithContext(ctx, req)
+}
+
+type TxoReference struct {
+	TxHash string
+	Index  uint32
+}
+
+func (u *UtxorpcClient) GetUtxosByRefs(
+	refs []TxoReference,
```

Comment:
For this, someone needs to know what the go-sdk only `TxoReference` is. This should reuse the `query.TxoRef` struct from UTxO RPC vs introduction of a new type.